CHANGELOG.md

```diff
@@ -15,6 +15,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Changed
 
+- Log messages are now consistent in style, and some have been removed.
```

Comment:
There should be no punctuation at the end of change log entries (see all other messages).